### ISSUE TITLE
Redundant "labels" field in alignment.py

### ISSUE THREAD

**Issue by thangld201** [zedian/hyperlora#3]
Hi @zedian, since the paper only applies loss on the hidden states of the sae and perturbed variant, I do not see why we need 'labels' field in alignment.py (L101-L110). Plus, this would cause error when passed to the RoBERTa model in `src/model_utils.py`. Could you re-check here ?

```
        [example["attention_mask"] for example in examples], dtype=torch.long
    )
    return {
        "original_embedding": original_embedding,
        "original_mask": original_mask,
        "input_ids": input_ids,
        "attention_mask": attention_mask,
        "dialect_features": dialect_feats[0],
        "labels": torch.zeros(len(examples))
    }
```
